app/scripts/controllers/noCitationsFound.ctrl.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+angular.module('yourStlCourts').controller('NoCitationsFoundCtrl', function (supportedMunicipalities) {
+  var ctrl = this;
+
+  ctrl.getSupportedMunicipalitiesList = function(){
+    return makeHtmlList(supportedMunicipalities)
+  };
+
+  function makeHtmlList(municipalityArray){
```

Comment:
If you're writing a function that has "html" in the name on a controller, it's inherently probably not doing what it should be doing in a controller. HTML construction should stay in the template as much as possible. You can do this in the HTML with `ng-repeat` and `$index` or `$last`: https://docs.angularjs.org/api/ng/directive/ngRepeat